### DISCUSSION TITLE
Integração com o slack do FrontendBrasil :grinning: :grinning:

### DISCUSSION THREAD

**Discussion by eduardojmatos** [frontendbr/forum#46] [Discussão]
À partir de agora temos integração com o Slack do FrontendBrasil! :raised_hands: :raised_hands: :raised_hands: 

Cada issue postada no forum vai aparecer no channel [#forum](https://frontendbrasil.slack.com/messages/forum) por lá!

Se você não se inscreveu no Slack FrontendBrasil, [visite aqui](http://frontendbrasil-slack.herokuapp.com/).

**Comment by napalmdev** [comment]
coloquei meu email lá, mas o e-mail de confirmação não chegou, isso já faz uns dias =/

**Comment by vitor-mendrone-deel** [comment]
Mano, que daora! :grin: Já entrei para o grupo!

**Comment by felipefialho** [comment]
Fantástico!

**Comment by vitorbritto** [comment]
Boa!

**Comment by willianjusten** [comment]
Boa =)
Integração é algo muito bom!

**Comment by suissa** [comment]
:+1:
